CI/run-perf-ci-suite

```diff
@@ -960,8 +967,8 @@ if ((! debugonly)) ; then
     fi
 
     if [[ -n "$artifactdir" && ! -d "$artifactdir" ]] ; then
-	exec > >(tee -a "$artifactdir/stdout.kata-perf-suite.log")
-	exec 2> >(tee -a "$artifactdir/stderr.kata-perf-suite.log" >&2)
+	exec > >(tee -a "$artifactdir/stdout.perf-ci-suite.log")
```

Comment:
Are we sure that removing the kata is necessary, or should we consider keeping it for future reference?

Reply:
It's just renaming it, not removing it.